src/metrics.go

```diff
@@ -79,10 +79,12 @@ func qCacheHitRatio(metrics map[string]interface{}) (float64, bool) {
 }
 
 func slaveRunningAsNumber(metrics map[string]interface{}) (int, bool) {
-	slaveRunning, ok := metrics["Slave_running"].(string)
-	if ok && slaveRunning == "ON" {
-		return 1, true
-	} else if ok && slaveRunning == "OFF" {
+	slaveIORunning, okIO := metrics["Slave_IO_Running"].(string)
+	slaveSQLRunning, okSQL := metrics["Slave_SQL_Running"].(string)
+	if okIO && okSQL {
+		if slaveIORunning == "Yes" && slaveSQLRunning == "Yes" {
+			return 1, true
+		}
 		return 0, true
 	}
 	return 0, false
```

Comment:
I would suggest to invert this `if` to reduce indentation and improve readability:

```suggestion
	if !okIO || !okSQL {
		return 0, false
	}

	if slaveIORunning == "Yes" && slaveSQLRunning == "Yes" {
		return 1, true
	}
	return 0, true
```

Reply:
Fixed. Thx!